src/Toml/Parser/Value.hs

```diff
@@ -22,9 +22,13 @@ import Control.Applicative.Combinators (between, count, option, optional, sepBy1
 import Data.Fixed (Pico)
 import Data.Time (Day, LocalTime (..), TimeOfDay, ZonedTime (..), fromGregorianValid,
                   makeTimeOfDayValid, minutesToTimeZone)
+
+import Data.String (fromString)
+
```

Comment:
Empty lines are redundant around this import 🙂  According to our style guide, imports should be grouped into external libraries, from the package itself, and `qualified` imports. And empty lines are only between those groups.

```suggestion
import Data.String (fromString)
```